### ISSUE TITLE
"detachKeys" in .docker/config.json breaks rocker

### ISSUE THREAD

**Issue by tomxtobin** [grammarly/rocker#77]
If the new `detachKeys` option is present in one's `.docker/config.json` file, `rocker build` breaks with the following error:
```
FATA[0000] json: cannot unmarshal string into Go value of type docker.dockerConfig 
```
I *think* this is an upstream bug with [fsouza/go-dockerclient](https://github.com/fsouza/go-dockerclient) but I'm not well-versed enough in Go to say for sure.

**Comment by romank87** [comment]
Will check that, thanks for reporting

**Comment by romank87** [comment]
@tomxtobin I sent pull request https://github.com/fsouza/go-dockerclient/pull/492 and it was accepted. Updated go-dockerclient already in one of our dev branches. Fixed version will be released soon.

**Comment by diunko** [comment]
updated go-dockerclient is included in 1.2.0 release.
